### PR TITLE
Teams: Update RSC concept doc

### PR DESCRIPTION
**Pull request by AkJo**

### CODE REVIEW

File: msteams-platform/graph-api/rsc/resource-specific-consent.md [hunk 1]
```diff
@@ -12,28 +12,28 @@ keywords: teams authorization OAuth SSO AAD rsc Graph
...
+Resource-specific consent (RSC) is a Microsoft Teams and Microsoft Graph API integration that enables your app to use API endpoints to manage specific resources—either teams or chats—within an organization. The resource-specific consent (RSC) permissions model enables *team owners* and *chat owners* to grant consent for an application to access and/or modify a team's data and a chat's data, respectively. The granular RSC permissions define what an application can do within a specific resource:
 
 ## Resource-specific permissions
 
 ### Resource-specific permissions for a team
 |Application permission| Action |
 | ----- | ----- |
-|TeamSettings.Read.Group | Get this team's settings.|
+|TeamSettings.Read.Group_ | Get this team's settings.|
```
Comment (jessiechan27) [inline msteams-platform/graph-api/rsc/resource-specific-consent.md:22]: What is this underline for?

**Review by jessiechan27 — APPROVED** [review]

File: msteams-platform/graph-api/rsc/resource-specific-consent.md (hunk 1 above)
Comment (AkJo) [reply msteams-platform/graph-api/rsc/resource-specific-consent.md:22]: good catch

Comment (subray2014) [inline msteams-platform/graph-api/rsc/resource-specific-consent.md:22]: What's the _ for?

Comment (AkJo) [reply msteams-platform/graph-api/rsc/resource-specific-consent.md:22]: it was a change that I was removing.

**Review by subray2014 — APPROVED** [review]

File: msteams-platform/graph-api/rsc/resource-specific-consent.md (hunk 1 above)
Comment (AkJo) [reply msteams-platform/graph-api/rsc/resource-specific-consent.md]: Result of an incomplete revert.

**Review by subray2014 — APPROVED** [review]

**Review by jessiechan27 — APPROVED** [review]

**Pull request merged**